### PR TITLE
#11 - append info to the new packages index

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #11 
Append package info when `Packages.gz`does not exist.

### CODE REVIEW

File: src/test/java/com/artipie/debian/metadata/PackageSimpleTest.java [hunk 1]
```diff
@@ -94,4 +80,49 @@ void addsPackagesItem() throws IOException {
...
+                    new ByteArrayInputStream(new BlockingStorage(asto).value(new Key.From(key)))
+                )
+            )
+        ) {
+            final ByteArrayOutputStream out = new ByteArrayOutputStream();
+            final byte[] buf = new byte[1024];
+            int cnt;
+            while (-1 != (cnt = gcis.read(buf))) {
+                out.write(buf, 0, cnt);
+            }
```
Comment (olegmoz) [inline src/test/java/com/artipie/debian/metadata/PackageSimpleTest.java:101]: @olenagerasimova I'd suggest to extract method for decompressing the archive and reading it as string, as it is quite big chunk of code duplicated

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, good to merge

**Pull request merged**